source/Server/Ldap/LdapContextProvider.cs

```diff
@@ -43,17 +43,22 @@ public LdapContext GetContext()
                     con.StartTls();
 
                 con.Bind(ldapConfiguration.Value.GetConnectUsername(), ldapConfiguration.Value.GetConnectPassword().Value);
-                con.Constraints.ReferralFollowing = ldapConfiguration.Value.GetReferralFollowingEnabled();
-                con.Constraints.HopLimit = ldapConfiguration.Value.GetReferralHopLimit();
-                con.Constraints.TimeLimit = ldapConfiguration.Value.GetConstraintTimeLimit() * 1000;
+
+                con.Constraints = new LdapConstraints(
```

Comment:
The property getter returns a clone - It's a trap!  These settings weren't taking effect at all 😬

Reply:
I have no idea why that would be a clone! Good work spotting that.